### ISSUE TITLE
[Feature Request] Set fees

### ISSUE THREAD

**Issue by shyfire131** [MutinyWallet/mutiny-web#283]
I tested the "swap to lightning" feature and missed the fact that it was using 1sat/vB fees. Not sure I'll see those sats again unless I go through RBF pain. Ability to adjust fees would be great. I think on-chain sends have the same issue.

p.s. is there a roadmap somewhere that I can check? Figured this might already be on it.

**Comment by benthecarman** [comment]
These are supposed to be automatically set, weird that it did not ever correctly sync the fee estimatss. Could you upload your logs?

**Comment by TonyGiorgio** [comment]
Yeah that's strange that the fee estimates were not coming in. We pull that from mempool by default. But yeah totally agree we need manual fee setting as an override option. 

We have a project board here: https://github.com/orgs/MutinyWallet/projects/1

**Comment by shyfire131** [comment]
Happy to upload logs. What's the process for that? DM them to someone on Matrix?

p.s. RIP, I see the tx is not RBF, so probably stuck in 1sat hell forever

**Comment by benthecarman** [comment]
You should be able to upload here. It is weird that it's not rbf. I thought we signaled for all of our txs.

**Comment by shyfire131** [comment]
No I mean that I'm not comfortable pasting the TxIDs in public since I consider that private information, but I'm happy to share them with your team directly.

**Comment by benthecarman** [comment]
Ah okay, yes dm them to me or Tony and we'll be sure to handle them privately

**Comment by benthecarman** [comment]
Looking into this, seems we actually don't signal rbf for channel opens. That should be easily fixable. 

Also according to mempool.space, txs are being purged under 9 sats/vbyte. In the dev tools there's a resync wallet button, if you click that it might get your funds back, because that tx is being purged.

**Comment by shyfire131** [comment]
Sent logs in DM.
Hah interesting @ purge. I'm still seeing the tx on mempool.space. If it's purged I should get "Transaction not found.
" on mempool.space right? Probably just need to wait a while and hold thumbs for more jpegs to be printed and that it does get purged eventually.

**Comment by benthecarman** [comment]
do you see the tx on blockstream.info or https://mempool.emzy.de/

If it's not on one of those you could switch your esplora url to one of these, then resync.

**Comment by TonyGiorgio** [comment]
I cut out all the details but this was interesting in the logs: 

```
Transaction details: TransactionDetails {
    fee: Some(
        5XXX,
    ),
}

...

Opening channel with params: ChannelOpenParams { sats_per_vbyte: 3X.X ... }

DEBUG [mutiny_core::onchain:423] Transaction details: TransactionDetails {
    fee: Some(
        2XX,
    ),
}
```

So it's trying to open channel with the correct sats per vbyte but then immediately creates a transaction with what looks like a 1 sat per byte.

**Comment by shyfire131** [comment]
> do you see the tx on blockstream.info or https://mempool.emzy.de/
> 
> If it's not on one of those you could switch your esplora url to one of these, then resync.

Cool, emzy.de did the trick! My on-chain balance looks good now, but I'm still seeing a stale balance in the Lightning card. Anyway to flush that apart from delete everything?

**Comment by TonyGiorgio** [comment]
Ah, yeah the lightning side of things may still think that the channel is in the process of opening while it checks for the transaction to confirm.

Since the transaction will never confirm, I'm actually not sure what it'll do. You could "close" the channel in the debug settings, but it might actually cause it to be CPFP'd. 

I think the reason you see the lightning balance is because we show the pending channel opens too.

**Comment by shyfire131** [comment]
No worries, I simply moved the funds to another wallet and nuked mutiny. I'll be back...

**Comment by futurepaul** [comment]
Once we land #396 can use that picker for fee selection
